src/main/java/gift/model/gift/Gift.java

```diff
@@ -11,19 +12,27 @@
 @Table(name = "gift")
 public class Gift {
 
+    @OneToMany(mappedBy = "gift", cascade = CascadeType.REMOVE)
+    protected List<Wish> wishes = new ArrayList<>();
```

Comment:
일반적인 테이블 설계와 마찬가지로 id를 가장 위로 올려두는 것이 좋을 것 같습니다ㅎㅎ
추가로 wishes도 protected로 선언되어 있네요!

```suggestion
    private List<Wish> wishes = new ArrayList<>();
```

Reply:
감사합니다! 놓친 부분이었네요.